pallets/stellar-relay/src/lib.rs

```diff
@@ -448,6 +467,30 @@ pub mod pallet {
 				Error::<T>::OrganizationLimitExceeded
 			);
 
+			let validators_old = Validators::<T>::get();
```

Comment:
At first, I thought that what you are doing here is wrong because of the weird naming. You are assigning the current validators to a variable that indicates it is the old validators. But they are not the old ones (yet). So I would rather call it `current_validators` or something.